components/EmbraceModal.vue

```diff
@@ -61,29 +54,42 @@
             dark
           />
         </div>
-        <div
-          class="button-white justify-center items-center mt-4 bg-white self-stretch flex gap-1 py-2 rounded-lg border border-gray-800 border-solid max-md:px-5"
+        <button
+          class="button-white text-lime-800 font-medium capitalize whitespace-nowrap justify-center items-center mt-4 bg-white self-stretch flex gap-1 py-2 rounded-lg border border-gray-800 border-solid max-md:px-5"
           @click.prevent.stop="copyText"
         >
           <img
+            v-if="!messageCopied"
             loading="lazy"
             src="https://cdn.builder.io/api/v1/image/assets/TEMP/d0e54027-9f1e-4d75-9014-63e737747321?"
             class="aspect-square object-contain object-center w-6 overflow-hidden shrink-0 max-w-full"
           >
-          <button
-            class="text-lime-800 text-center text-base font-medium capitalize whitespace-nowrap"
+          <span v-if="!messageCopied">
+            {{ embracePage?.data.copy_text_button_label || 'Copy Text' }}
+          </span>
+          <img
+            v-if="messageCopied"
+            :class="`${
+              dark ? 'w-6 h-6' : 'bg-sushi-100 w-6 h-6'
```

Comment:
dark as a variable doesn't exist? Don't think we need to differentiate here between dark or not since it's a very specific application here and not reusable component